src/components/BaseMap/BaseMap.tsx

```
@@ -1,24 +1,42 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import * as maptilersdk from "@maptiler/sdk";
 
-import Map from "react-map-gl/maplibre";
+import Map, { Layer, Source } from "react-map-gl/maplibre";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)